model-optimizer/extensions/ops/ReduceOps.py

```diff
@@ -138,11 +145,30 @@ class ReduceMean(ReduceOp):
     enabled = True
 
 
-class ReduceL2(ReduceOp):
-    op = 'ReduceL2'
-    op_type = None
+class AttributedReduceLp(ReduceOp):
+    op = 'AttributedReduceLp'
+    op_type = 'AttributedReduceLp'
+    enabled = True
+
+    def __init__(self, graph: Graph, attrs: dict):
+        op_attrs = {'version': 'opset4',
```

Comment:
Let's add an assert statements to the constructor to check that both `axis` and `p` attributes are not `None`.

Reply:
Done.